containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -37,24 +37,29 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
 
-### Scaleway Kubernetes products 
+### Scaleway Kubernetes products
 
-| Kubernetes version | Official release | Official End of Life | Release                      | Deprecation      | End of Support    |
+| Kubernetes version | Official release | Official End of Life | Release                      | Deprecation      | End of support    |
 |--------------------|------------------|----------------------|------------------------------|------------------|-------------------|
-| 1.19               | August 26, 2020  | October 2021         | August 31, 2020              | August 2022      | October 15, 2022  |
-| 1.20               | December 8, 2020 | February 2022        | December 8, 2020             | November 2022    | January 15, 2023  |
-| 1.21               | April 8, 2021    | June 2022            | April 30, 2021               | April 2023       | June 15, 2023     |
-| 1.22               | August 4, 2021   | October 2022         | August 11, 2021              | August 2023      | October 15, 2023  |
-| 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
-| 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
-| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.27               | April 2023       | June 2024            | April 2023                   | April 2025       | 15 May 2025       |
 | 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 15, 2025    |
+| 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
+| 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
+| 1.22               | August 4, 2021   | October 2022         | August 11, 2021              | August 2023      | October 15, 2023  |
+| 1.21               | April 8, 2021    | June 2022            | April 30, 2021               | April 2023       | June 15, 2023     |
 
 ## Version information
 For a complete changelog update, refer to the official [Kubernetes changelog](https://kubernetes.io/releases/).
 
 The following documentation lists only the main feature changes for each version and provides a link to the version release announcement on Kubernetes' official blog.
 
+### 1.27
+* Official [release announcement](https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/)
+
+**Main changes:**
+* This release consists of 60 enhancements. 18 of those enhancements are entering Alpha, 29 are graduating to beta, and 13 are graduating to stable.
```

Comment:
IMO, Alpha/Beta is either in upper or lowercase

Reply:
```suggestion
* This release consists of 60 enhancements. 18 of those enhancements are entering alpha, 29 are graduating to beta, and 13 are graduating to stable.
```